### PR TITLE
Making headings consistent

### PR DESCRIPTION
**Pull request by richsalz**
Also "2." was already at level-one.

### CODE REVIEW

**Comment by nllz** [comment]
Does this not interfere with Wagtail @ghwood ?

**Comment by richsalz** [comment]
If it does interfer, I'll change this to just fix inconsistencies.

**Comment by richsalz** [comment]
markdownlint complains.  So I put back all the "extra" indents.  I left `# About ...` instead of `## About...` so @ghwood please tell me if that needs fixing.  All other consistency changes remain.

File: Tao.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ document but an informal and informational overview.
 
 ## Table of Contents
 
-1. <a href="#introduction">Introduction</a><br>
+1 <a href="#introduction">Introduction</a><br>
```
Comment (larseggert) [inline Tao.md:32]: More readable:

```suggestion
1 [Introduction](#introduction)<br>
```

Likewise for other links.

Comment (richsalz) [reply Tao.md:32]: I would rather fix content and leave the markdown fixups for a separate PR.  See #42

Comment (larseggert) [reply Tao.md:32]: ACK

**Pull request merged**